README.md

```diff
@@ -62,16 +62,16 @@ I wanted to avoid handling video URLs since that's its own project and didn't wa
 Here's my very "scientific" comparison of `skonaki` with other tools.
 I did not bother with those that required signing up or had a paywall:
 
-* [youtubesummarizer.com](https://youtubesummarizer.com/)
-  * Works pretty well, there are some ads and I like the executive summary part that they also provide. I'd say the quality of the results is similar to `skonaki`.
-* [summarize.tech](https://www.summarize.tech/)
-  * Wasn't very impressed with the results. It summarizes OKish but wouldn't be good enough for studying purposes.
-* [notegpt.io](https://notegpt.io/)
-  * Rather quick, I only tried their web version, while it also comes as a plugin. While the summary was on point, I found it too short to use for studying purposes. Great for summarizing though.
+- [youtubesummarizer.com](https://youtubesummarizer.com/)
+  - Works pretty well, there are some ads and I like the executive summary part that they also provide. I'd say the quality of the results is similar to `skonaki`.
+- [summarize.tech](https://www.summarize.tech/)
+  - Wasn't very impressed with the results. It summarizes OKish but wouldn't be good enough for studying purposes.
+- [notegpt.io](https://notegpt.io/)
+  - Rather quick, I only tried their web version, while it also comes as a plugin. While the summary was on point, I found it too short to use for studying purposes. Great for summarizing though.
 
 | Tool                  | Summaries | Timestamps | Customization | Notes                                                                                      |
 | --------------------- | --------- | ---------- | ------------- | ------------------------------------------------------------------------------------------ |
-| `skonaki`             | 🏆         | ✅          | ✅             | Can customize anything from the prompt used, to how often a "summary" is produced          |
-| youtubesummarizer.com | 🏆         | ❌          | 😒             | There's a sliding scale for how long the summary should be, but doesn't get too "detailed" |
-| summarize.tech        | 😒         | ✅          | ❌             | No customization, their summary is not very suitable for studying                          |
-| notegpt.io            | 😒         | ❌          | ❌             | The summary is pretty good, but not detailed enough for studying                           |
+| `skonaki`             | 🏆        | ✅         | ✅            | Can customize anything from the prompt used, to how often a "summary" is produced          |
+| youtubesummarizer.com | 🏆        | ❌         | 😒            | There's a sliding scale for how long the summary should be, but doesn't get too "detailed" |
+| summarize.tech        | 😒        | ✅         | ❌            | No customization, their summary is not very suitable for studying                          |
+| notegpt.io            | 😒        | ❌         | ❌            | The summary is pretty good, but not detailed enough for studying                           |
```

Comment:
lol can you undo the formatter changes? :grin: They were probably automatically done by a tool

Reply:
`ctrl+shift+s` to save without auto-formatting in case you use VS Code :grin:

Reply:
Oops - yes, they should now be undone :laughing: